### ISSUE TITLE
Screen reader focus is moving to entire content present on the App and announcing as group.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27878]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: [Telegram: Contact @SCTejTestbot](https://t.me/SCTejTestbot) on edge.
3. SCTestbot home screen will appear.
4. Turn on Narrator and navigate the page using tab key.
5. Observe an issue that Screen reader focus in moving to entire content present on the App and announcing as group.

### Expected behaviour

Screen reader should not announce group for every element present on the page, focus should move to individual interactive controls and screen reader should announce proper role, name and state of the interactive control while navigating using tab key.

### Actual behaviour

Screen reader focus is moving to entire content present on the App and announcing as group.

https://github.com/telegramdesktop/tdesktop/assets/93735775/24a4b8fd-515b-4bda-863e-b80b5a066727

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
